tsconfig.json

```diff
@@ -1,15 +1,14 @@
 {
     "compilerOptions": {
-      "target": "es5",
+      "target": "es2021",
+      "module": "ESNext",
       "allowJs": false,
       "strict": true,
       "noEmit": true,
       "esModuleInterop": true,
       "moduleResolution": "node",
-      "resolveJsonModule": true,
-      "isolatedModules": true,
       "incremental": true,
-      "downlevelIteration": true
```

Comment:
These were removed recently from the closed source config I had originally copied this file from.